src/tooltip/index.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+class Tooltip extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      showTooltip: false,
+      xPosition: 0,
+      yPosition: 0
+    };
+    this.handleRenderTooltip = this.handleRenderTooltip.bind(this);
+  }
+
+  shouldComponentUpdate (nextProps, nextState) {
+    const { showTooltip } = this.state;
+    if (nextState.showTooltip === showTooltip) {
+      return false;
+    }
+    return true;
+  }
+
+  handleRenderTooltip (event) {
+    const { showTooltip } = this.state;
+    const xPosition = event.clientX;
+    const yPosition = event.clientY;
+    this.setState({ showTooltip: !showTooltip,
+      xPosition,
```

Comment:
I managed to cause issues where coordinates didn't have time to update, resulting in tooltip showing up over the wrong cell. I think the tooltip position should be fixed, centered above the value.